index.d.ts

```diff
@@ -19,6 +19,14 @@ export interface Options {
    * @default process.cwd()
    */
   repo?: string
+
+  /**
+   * Path(s) to a file or directory to limit results to specific changes. When no
+   * `path` is specified, the entire history of the project id returned.
```

Comment:
I think that it's better to leave word wrapping up to the editor since different contributors might have different widths of their editors:

```suggestion
   * Path(s) to a file or directory to limit results to specific changes. When no `path` is specified, the entire history of the project id returned.
```